src/python_testing/matter_testing_infrastructure/chip/testing/matter_testing.py

```diff
@@ -612,6 +612,142 @@ def test_skipped(self, filename: str, name: str):
         logging.info(f"Skipping test from {filename}: {name}")
 
 
+class PIXITType(Enum):
+    INT = "int"
```

Comment:
This approach in general is OK, but I do want to consider this in the larger context of verifying that the test invocations are correct in a broader way. We can land this in stages, but it is important to consider how the other checks are going to be integrated here.

In particular, test authors need an easy way to declare certain other test requirements that also need verification. Ex. Endpoint https://github.com/project-chip/matter-test-scripts/issues/450, which we should consider mandating with no default for tests that are run against a single endpoint, no --endpoint flag for tests that run against the entire node at once. Need a way to declare that a test requires a commissioned device, whether or not that commissioning happens as a part of the test invocation. Some tests require passcodes, discriminators etc, even for tests where the commissioning happens external to the test. Tests that DO use PICS (hopefully fewer and fewer) should have a PICS file specified (and this get extra difficult in the context of the test harness)

Anyway, worth thinking through how these extended checks would fit in with this design. I don't see anything in particular blocking these options right now.

It's also worth thinking through how to expose this properly to test authors. ie, is there a way that we can alert test authors that are using test flags that they should be providing these functions?